sdk/identity/azure-identity/src/test/java/com/azure/identity/implementation/IdentityClientTests.java

```diff
@@ -222,99 +260,111 @@ public void testValidIdentityEndpointMSICodeFlow() throws Exception {
             .setManagedIdentityType(ManagedIdentityType.APP_SERVICE)
             .setManagedIdentityParameters(new ManagedIdentityParameters()
                 .setIdentityEndpoint(endpoint)
-                .setIdentityHeader(secret));
+                .setIdentityHeader(secret))
+            .setConfiguration(configuration);
```

Comment:
indentation (and the next few too)

Reply:
This is the correct indentation as the deeper indented lines are creating an Object which closes before setting configuration

Reply:
Oh I missed the second constructor, my bad.